bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,338 @@
+            return
+
+        await ctx.send(
+            f"The word ~~is~~ was **{self.word}**. The poll has already started, so you cannot "
+            "add an entry."
+        )
+
+    # @in_month(month_to_limit_to)
+    @spook_name_rate.command(name="add", aliases=["+", "register"])
+    async def add_word(self, ctx: Context, *, word: str) -> None:
+        """A command that adds your word!"""
+        if self.poll:
+            logger.info(f"{ctx.message.author} tried to add a word, but the poll had already started.")
+            await ctx.send("Sorry, the poll has started! You can try and participate in the next round though!")
+            return
+
+        message = ctx.message  # get the message
+
+        for message_id in self.messages:
+            data = self.messages.get(message_id)
+            if data['author'] == message.author:  # if the author has already added an entry
+                await ctx.send(
+                    "But you have already added an entry! Type "
+                    f"`{self.bot.command_prefix}spooknamerate "
+                    "delete` to delete it, and then you can add it again"
+                )
+                return
+
+            elif data['word'] == word:
+                # Someone has added the word
+                await ctx.send("TOO LATE. Someone has already added this word.")
+                return
+
+        self.messages[message.id] = {  # store it
+            'word': word,
+            'author': message.author,
+            'score': 0,
+        }
+
+        for emoji in self.emojis_val:
+            await message.add_reaction(emoji)  # add each emoji
+
+        logger.info(f"{message.author} added the word {word!r}")
+
+        await ctx.send(f"{word!r} added successfully!\n{random.choice(self.added_messages)}")
+
+    # @in_month(month_to_limit_to)
+    @Cog.listener()
+    async def on_reaction_add(self, reaction: Reaction, user: User) -> None:
+        """Ensures that each user adds one and only one reaction."""
+        async with self.checking_messages:  # Acquire the lock so that the dictionary isn't reset while iterating.
+            if reaction.emoji in self.emojis_val and reaction.message.id in self.messages:
+                msg: Message = reaction.message
+                # create a custom counter
+                reaction_counter = defaultdict(int)
+                for r in msg.reactions:
+                    async for reaction_user in r.users():
+                        if reaction_user == self.bot.user:
+                            continue
+                        reaction_counter[reaction_user] += 1
+
+                if reaction_counter[user] > 1 and user != self.bot.user:
+                    await user.send("Sorry, you have already added a reaction, please remove your reaction and try "
+                                    "again")
+                    return await reaction.remove(user)  # removes user's reaction
+
+    # @in_month(month_to_limit_to)
```

Comment:
Please remove these if the cogcheck works

Reply:
Ah I forgot to remove that